libs/libpcf/example/example_xt/fpga_io_location.xml

```diff
@@ -0,0 +1,72 @@
+<!--
+	- FPGA Fabric I/O Information
+	- Generated by OpenFPGA
+	- Date: Wed Dec 17 09:48:40 2025
+-->
+
+<io_coordinates>
```

Comment:
I have checked your unit test ``test_pcf2bitstream_setting.cpp``. It does not require this file as an input. It not, just remove it from the PR.